ui/admin/app/styles/_notify.scss

```diff
@@ -86,7 +86,7 @@
   }
 }
 
-@media only screen and (max-width: 680px) {
```

Comment:
Was this a manual change or an automated one? if manual, can you elaborate on why we change?

Reply:
Why we are performing this change within `ui/admin` if this is desktop upgrade?

Reply:
@calcaide updating the `stylelint-standard-config-scss` to the correct version that matches with `stylelint` and not give us an unmet peer dependency, has a new modern way of writing media query ranges.  `prefix` allows the use of patterns like `min-width` where as `context` wants more the modern take which is using an evaluator to express it.

https://stylelint.io/user-guide/rules/media-feature-range-notation/

Reply:
@calcaide I will create a ticket to upgrade both `stylelint-standard-config-scss` dependencies for `admin` and `desktop`.  May also be a discussion point for #tech-time, because using `context` patterns for media queries is a pattern change.